pymtl3/dsl/test/sim_utils.py

```diff
@@ -497,3 +498,14 @@ def cleanup_signals( m ):
             cleanup_signals( obj )
 
   cleanup_signals( s )
+
+  def create_reset( top ):
+    def reset():
+      print()
```

Comment:
why do we need a print here?

Reply:
We need this print for the first line in pytest

Reply:
I think it is fine to add a print() in each test case. If reset is called again during the simulation this will also print an empty line.